site/en/posts/bluetooth-microphone-super-resolution-better-nbs-sound-quality-through-machine-learning.md

```diff
@@ -0,0 +1,87 @@
+
+!!! aside.message--note
+**Note:** See the papers [U-Net: Convolutional Networks for Biomedical Image Segmentation](https://arxiv.org/abs/1505.04597) and [Generative Adversarial Nets](https://arxiv.org/pdf/1406.2661.pdf) for more details on SEANet, U-Net, and adversarial training.
+!!!
+
+To train the model, we need paired audio signals: audio from a Bluetooth NBS connection and high-quality target audio. We used software simulation to create audio signals that resembled what we would get from a Bluetooth NBS connection. The Continuously Variable Slope Delta (CVSD) codec, used in Bluetooth NBS connection, was incorporated into the simulation process to model the distribution of audio signals. In addition to removing the high-frequency components in the audio signal, we also suppressed the frequency components below 510 Hz in a stochastic manner to mimic the recommended Bluetooth frequency mask.
+
+The final processing steps were:
+
+1.  **Augmentation**: The input audio signal was augmented with random cropping, gain adjustment, band-pass filtering, and Gaussian noise.
+1.  **Downsampling**: Audio was down-sampled to 8 kHz; the sampling rate of narrow-band speech.
+1.  **Encoding and decoding**: The CVSD codec was added to make the audio signal distribution similar to that from narrow-band speech connection.
+1.  **Upsampling**: Finally, the audio signal was upsampled to the target sampling rate before feeding into the model. Note that the upsampling here does not reconstruct high-frequency components; instead, it fills the high-frequency bins with aliases of low-frequency bins or simply with zeros.
+
+We also collected audio samples from different Bluetooth microphones for model evaluation, selecting models based on the evaluation results.
+
+## Results preferred by over 80% of users
+
+We presented users with two versions of an audio clip recorded on Airpods: one before processing and one after processing. Our model was able to reconstruct high frequency components generating a sound quality preferred by 82.8% of 450 users.
+
+Listen to the below before and after demo samples processed by the model.
+
+| Audio signal from Bluetooth NBS connections                                                                                | Audio signal processed by the model                                                                                                            |
+| -------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------- |
+| #1[From the spectrogram, we can see that the highest frequency is capped at 4 kHz.](ix://posts/bluetooth-nbs/Inline-2.png) | #2[The highest frequency becomes 12 kHz and some of the higher frequency components are reconstructed.](ix://posts/bluetooth-nbs/Inline-3.png) |
```

Comment:
If we're going to do these as a table, they don't need to be figures, and the links to download the audio files have been lost in this conversion.